src/pybind/mgr/dashboard/frontend/e2e/page-helper.po.ts

```diff
@@ -104,6 +104,27 @@ export abstract class PageHelper {
     return element.all(by.cssContainingText('.datatable-body-cell-label', content)).first();
   }
 
+  getFirstExpandCollapseElement(): ElementFinder {
+    return element.all(by.className('tc_expand-collapse')).first();
+  }
+
+  getExpandCollapseElement(elem: ElementFinder): ElementFinder {
```

Comment:
I could use one of those return statements instead
```
getExpandCollapseElement(content: string): ElementFinder {
    return this.getTableRow(content).element(by.className('tc_expand-collapse'));
    return element(by.cssContainingText('.datatable-body-row', content)).element(by.className('tc_expand-collapse'));
}
```
You would prefer the first one, correct?

Reply:
Depends if `getTableRow` is only used in that method. I would prefer something like `const tableRow = element(by.cssContainingText('.datatable-body-row', content))` before the return statement, so that it's clearer what is done.